android/activitysources/src/main/java/com/fjuul/sdk/activitysources/entities/ActivitySourcesManager.java

```diff
@@ -395,15 +396,23 @@ private void configureGoogleFitState(@Nullable List<TrackerConnection> trackerCo
         } else {
             googleFit.setLowerDateBoundary(null);
         }
-    }
 
-    private void configureHealthConnectState(@Nullable List<TrackerConnection> trackerConnections) {
         final TrackerConnection hcTrackerConnection = Optional.ofNullable(trackerConnections)
             .flatMap(connections -> connections.stream()
                 .filter(c -> c.getTracker().equals(TrackerValue.HEALTH_CONNECT.getValue()))
                 .findFirst())
             .orElse(null);
 
+        if (hcTrackerConnection != null) {
```

Comment:
This method is now called `configureGoogleFitState` but handles both Google Fit and Health Connect - should be renamed